build_che.sh

```diff
@@ -16,7 +16,9 @@ scl enable rh-maven33 'mvn -B clean install'
 cd ..
 
 git clone -b ${GIT_BRANCH} ${GIT_REPO}
-cd che 
+cd che
+# Use our modified stacks
+mv ./ide/che-core-ide-stacks/src/main/resources/stacks.json.centos \ 
+ ./ide/che-core-ide-stacks/src/main/resources/stacks.json
```

Comment:
This script is for jenkins job usage only so a developer should not use it.
`che-core-ide-stacks` component is already build when we build from the root `che` directory isn't it?